### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by rytilahti** [happyleavesaoc/python-firetv#17]
Hi,

it would be nice to have a new release for pypi after #11 gets merged in, thanks!

**Comment by happyleavesaoc** [comment]
pypi now has `1.0.3`. Install with `pip install firetv[firetv-server] --process-dependency-links`

Let me know if that doesn't work.

**Comment by Dirt-Nasty** [comment]
@happyleavesaoc I'm getting this error when running the install, http://pastebin.com/Gd1rrX5b

**Comment by happyleavesaoc** [comment]
@Dirt-Nasty What version of python, and what version of pip?

**Comment by Dirt-Nasty** [comment]
@happyleavesaoc 

pip 1.5.6
Python 2.7.9

Getting this now...

`  Could not find a version that satisfies the requirement adb==1.1.0+git.master (from firetv[firetv-server]) (from versions: 1.1.1)
No matching distribution found for adb==1.1.0+git.master (from firetv[firetv-server])
`

**Comment by TheLerch** [comment]
I'm getting the same error.

```
root@cm0auto00:/home/pi# pip install firetv[firetv-server] --process-dependency-links
Collecting firetv[firetv-server]
  Using cached firetv-1.0.3-py2-none-any.whl
Collecting adb==1.1.0+git.master (from firetv[firetv-server])
  Could not find a version that satisfies the requirement adb==1.1.0+git.master (from firetv[firetv-server]) (from versions: 1.1.1)
No matching distribution found for adb==1.1.0+git.master (from firetv[firetv-server])

```

Pip and Python version on Raspian.

pip 9.0.1 from /usr/local/lib/python2.7/dist-packages (python 2.7)

**Comment by TheLerch** [comment]
I think it's doing a specific check for adb v1.1.0 where the latest is adb-1.1.1.

Let me see if I can force it to install the older version of adb.

**Comment by TheLerch** [comment]
I was able to install adb 1.1.0 from github and still had the same error.

I've gone back to the regular pip installed version now.

**Comment by TheLerch** [comment]
I could not get the requirment to install even from Git so I forked a version, modified the requirements to the latest adb (1.1.1) and it did install for me.

root@cm0auto00:~# pip install git+https://github.com/llabas/python-firetv
Collecting git+https://github.com/llabas/python-firetv
  Cloning https://github.com/llabas/python-firetv to /tmp/pip-GyBQwL-build
Collecting adb==1.1.1 (from firetv==1.0.3)
Requirement already satisfied: python-gflags in /usr/local/lib/python2.7/dist-packages (from adb==1.1.1->firetv==1.0.3)
Requirement already satisfied: libusb1 in /usr/local/lib/python2.7/dist-packages (from adb==1.1.1->firetv==1.0.3)
Requirement already satisfied: M2Crypto in /usr/local/lib/python2.7/dist-packages (from adb==1.1.1->firetv==1.0.3)
Requirement already satisfied: typing in /usr/local/lib/python2.7/dist-packages (from M2Crypto->adb==1.1.1->firetv==1.0.3)
Installing collected packages: adb, firetv
  Found existing installation: adb 1.1.0
    Uninstalling adb-1.1.0:
      Successfully uninstalled adb-1.1.0
  Running setup.py install for firetv ... done
Successfully installed adb-1.1.1 firetv-1.0.3

Also needed:

 pip install PyYAML

So far it's working.  The changes I made in my fork are here:

https://github.com/llabas/python-firetv/commit/58305f2f2453a5902aec655761ea325d2c609fcd

**Comment by TheLerch** [comment]
New version install via pip works and 1.04 is installed on Raspbian.

**Comment by rytilahti** [comment]
This is now stale so closing it.

**Comment by kzaoaai** [comment]
@llabas why are we not funding this 💃 After several hours trying to fix the setup, your fork worked for me as-is, thanks!
